scripts/deploy/package-webapi.ps1

```diff
@@ -37,7 +45,7 @@ if (!(Test-Path $publishOutputDirectory)) {
 }
 
 Write-Host "Build configuration: $BuildConfiguration"
-dotnet publish "$PSScriptRoot/../../webapi/CopilotChatWebApi.csproj" --configuration $BuildConfiguration --framework $DotNetFramework --runtime $TargetRuntime --self-contained --output "$publishOutputDirectory"
+dotnet publish "$PSScriptRoot/../../webapi/CopilotChatWebApi.csproj" --configuration $BuildConfiguration --framework $DotNetFramework --runtime $TargetRuntime --self-contained --output "$publishOutputDirectory" /p:AssemblyVersion=$Version /p:FileVersion=$Version /p:InformationalVersion=$InformationalVersion
```

Comment:
why do we need them both if they hold the same value?

Reply:
If we don't specify either, it will default to 1.0.0 and just be confusing:
https://learn.microsoft.com/en-us/troubleshoot/developer/visualstudio/general/assembly-version-assembly-file-version